tests/workspace/__snapshots__/test_manager.ambr

```diff
@@ -181,6 +181,13 @@
   '''
   definition_version: '2'
   entities:
+    app:
+      from:
+        target: pkg
+      identifier: myapp_app
+      meta:
+        warehouse: app_wh
+      type: application
```

Comment:
Kind of a nit, but can we control the order of the keys? Specifically I'd prefer to have "type" to be first and "identifier" second, but if all of the keys could be in the same order as the schema it would be the best.

Reply:
nah, this is always rendered in alphabetical order no matter the order it's added in the dict